dev/src/reference.ts

```diff
@@ -1048,7 +1048,9 @@ export class Query {
         const fieldValue = documentSnapshot.get(fieldOrder.field);
         if (fieldValue === undefined) {
           throw new Error(
-            `Field "${fieldOrder.field}" is missing in the provided DocumentSnapshot. ` +
+            `Field "${
+              fieldOrder.field
+            }" is missing in the provided DocumentSnapshot. ` +
```

Comment:
unrelated?

Reply:
Yes. I guess prettier got updated.

Reply:
Had to reinstall prettier but it's gone now.